mongo/options/mongooptions.go

```diff
@@ -131,12 +133,21 @@ func (af *ArrayFilters) ToArray() ([]bson.Raw, error) {
 		registry = bson.DefaultRegistry
 	}
 	filters := make([]bson.Raw, 0, len(af.Filters))
+	buf := new(bytes.Buffer)
+	enc := new(bson.Encoder)
```

Comment:
We should initialize these values in the test case funcs.

Reply:
Will skip this as it is not in a test.

Reply:
Ah, good catch. Please disregard.